apps/tad/base/deployment-pgadmin.yaml

```diff
@@ -12,15 +12,21 @@ spec:
       automountServiceAccountToken: false
       securityContext:
         fsGroup: 5050
-      initContainers:
-        - name: init-clean-sessions
+      containers:
+        - name: clean-sessions
           image: busybox
           command: ['sh', '-c', 'rm -rf /var/lib/pgadmin/sessions']
           volumeMounts:
             - name: pgadmin-data
               mountPath: /var/lib/pgadmin
-
-      containers:
+          resources:
+            limits:
+              cpu: 100m
```

Comment:
I would leave the limit on CPU out, as CPUs is not really limited over time, extra resource: https://loft.sh/blog/how-to-set-up-kubernetes-requests-and-limits/

Reply:
see https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/ the official documentation why i would set it